setup.cfg

```diff
@@ -4,3 +4,7 @@ with-coverage=1
 cover-package=baseplate
 cover-html=1
 cover-html-dir=build/coverage
+
+[flake8]
+ignore = E501,E128,E226
```

Comment:
(note these ignores aren't new)

i'd like to get 501 going, that's a good one. the three outstanding long-lines when setting max length to 100 are all URLs in comments though :(